From 828873b10e74ac871fa833840362d7e893b352b3 Mon Sep 17 00:00:00 2001
From: Krzysztof Chojnowski <krzysiek@embedev.pl>
Date: Thu, 9 Apr 2020 13:20:33 +0200
Subject: [PATCH] Load the kernel and dtb from fat partition

---
 include/configs/somlabs_visionsom_6ull.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/include/configs/somlabs_visionsom_6ull.h b/include/configs/somlabs_visionsom_6ull.h
index a26d4134c0..04f191add5 100644
--- a/include/configs/somlabs_visionsom_6ull.h
+++ b/include/configs/somlabs_visionsom_6ull.h
@@ -62,18 +62,18 @@
 	"script=boot.scr\0" \
 	"image=zImage\0" \
 	"splashimage=0x80000000\0" \
-	"splashfile=/boot/splash.bmp\0" \
+	"splashfile=splash.bmp\0" \
 	"setrootnand=setenv rootspec root=ubi0:rootfs ubi.mtd=ubi rootfstype=ubifs "MFG_NAND_PARTITION"\0" \
 	"setbootscriptnand=setenv loadbootscript ubifsload ${loadaddr} /boot/${script};\0" \
 	"setloadnand=setenv loadimage ubifsload ${loadaddr} /boot/${image}; " \
 	            "setenv loadfdt ubifsload ${fdt_addr} /boot/${fdt_file};\0"\
 	"mmcdev=0\0" \
 	"mmcpart=1\0" \
-	"mmcroot=/dev/mmcblk1p1 rootwait rw\0" \
+	"mmcroot=/dev/mmcblk1p2 rootwait rw\0" \
 	"setrootmmc=setenv rootspec root=${mmcroot}\0" \
-	"setbootscriptmmc=setenv loadbootscript ext4load mmc ${mmcdev}:${mmcpart} ${loadaddr} /boot/${script};\0" \
-	"setloadmmc=setenv loadimage ext4load mmc ${mmcdev}:${mmcpart} ${loadaddr} /boot/${image}; " \
-	           "setenv loadfdt ext4load mmc ${mmcdev}:${mmcpart} ${fdt_addr} /boot/${fdt_file};\0" \
+	"setbootscriptmmc=setenv loadbootscript fatload mmc ${mmcdev}:${mmcpart} ${loadaddr} ${script};\0" \
+	"setloadmmc=setenv loadimage fatload mmc ${mmcdev}:${mmcpart} ${loadaddr} ${image}; " \
+	           "setenv loadfdt fatload mmc ${mmcdev}:${mmcpart} ${fdt_addr} ${fdt_file};\0" \
 	"setbootargs=setenv bootargs console=${console},${baudrate} " \
 		BOOTARGS_CMA_SIZE \
 		"${rootspec}\0" \
